src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -189,11 +223,9 @@ ucs_status_t uct_ib_mlx5_txwq_init(uct_priv_worker_t *worker,
         return UCS_ERR_IO_ERROR;
     }
 
-    ucs_debug("tx wq %d bytes [bb=%d, nwqe=%d]",
-              qp_info.dv.sq.stride * qp_info.dv.sq.wqe_cnt,
-              qp_info.dv.sq.stride, qp_info.dv.sq.wqe_cnt);
-
-    if (qp_info.dv.bf.size > 0) {
+    if (cfg_mmio_mode != UCT_IB_MLX5_MMIO_MODE_AUTO) {
+        mmio_mode = cfg_mmio_mode;
```

Comment:
may be print an error if one of bf methods is set, but `bf.size` is `0`? and set `db` mode then

Reply:
I prefer not to, since the goal of this parameter is to bypass the existing logic (it's not intended for regular use case, but as a tweak in case something goes wrong in user setup)